ERROR: test1_load_dbpedia (__main__.TestSparqlStore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/michele.pasin/Dropbox/code/python/ontospy/ontospyProject/ontospy/tests/test_sparql.py", line 38, in test1_load_dbpedia
    o = Ontospy(sparql_endpoint=self.ENDPOINT, verbose=True)
  File "/Users/michele.pasin/Dropbox/code/python/ontospy/ontospyProject/ontospy/core/ontospy.py", line 91, in __init__
    self.load_sparql(sparql_endpoint, verbose, hide_base_schemas, credentials)
  File "/Users/michele.pasin/Dropbox/code/python/ontospy/ontospyProject/ontospy/core/ontospy.py", line 127, in load_sparql
    graph = rdflib.ConjunctiveGraph('SPARQLUpdateStore')
  File "/Users/michele.pasin/Envs/ontospy_py3/lib/python3.7/site-packages/rdflib/graph.py", line 1304, in __init__
    super(ConjunctiveGraph, self).__init__(store, identifier=identifier)
  File "/Users/michele.pasin/Envs/ontospy_py3/lib/python3.7/site-packages/rdflib/graph.py", line 313, in __init__
    self.__store = store = plugin.get(store, Store)()
  File "/Users/michele.pasin/Envs/ontospy_py3/lib/python3.7/site-packages/rdflib/plugins/stores/sparqlstore.py", line 606, in __init__
    **kwds
  File "/Users/michele.pasin/Envs/ontospy_py3/lib/python3.7/site-packages/rdflib/plugins/stores/sparqlstore.py", line 232, in __init__
    self.setUseKeepAlive()
  File "/Users/michele.pasin/Envs/ontospy_py3/lib/python3.7/site-packages/SPARQLWrapper/Wrapper.py", line 587, in setUseKeepAlive
    if urllib2._opener and any(isinstance(h, HTTPHandler) for h in urllib2._opener.handlers):
NameError: name 'urllib2' is not defined